src/sagas/wallet.js

```diff
@@ -250,6 +252,13 @@ export function* startWallet(action) {
     });
     console.log('[+] Start wallet.', serverInfo);
 
+    // Use HTR as default
+    let nativeToken = hathorLibConstants.DEFAULT_TOKEN_CONFIG;
```

Comment:
If we can't use the method from storage, we should fix this, to use `DEFAULT_NATIVE_TOKEN_CONFIG`